.gitignore

```diff
@@ -40,12 +40,14 @@
 /docs/api/
```

Comment:
are there folders to unignore after removing miredot?

Reply:
Possibly, would it be cleaner if we outputted the documentation in the same location?

Reply:
i'm not a fan of how miredot spread all the generated files in each of the packages so i would prefer if we could output all the docs into the same directory if possible